lib/css/modules/_accountSwitcher.scss

```diff
@@ -1,8 +1,10 @@
 #RESAccountSwitcherDropdown {
-	min-width: 110px;
-	width: auto;
 	display: none;
+	height: calc(100vh - 80px);
+	min-width: 110px;
+	overflow: scroll;
```

Comment:
This forces scrollbars to always display. It should be `auto`, or, for completeness:

```css
overflow-x: hidden;
overflow-y: auto;
```